Cargo.toml

```diff
@@ -44,13 +44,12 @@ base64 = "0.10"
 env_logger =  {version = "0.6", default-features = false, features = ["termcolor","humantime","atty"]} 
 futures = "0.1"
 getopts = "0.2"
-hyper = "0.11"
 log = "0.4"
 num-bigint = "0.2"
 protobuf = "2.8.1"
 rand = "0.7"
 rpassword = "3.0"
-tokio-core = "0.1"
+tokio = "0.1"
```

Comment:
```
tokio = { version = "0.2", features = ["full"] } 
```

Reply:
I seem to have made a mistake while rebasing my old branch on dev, let me give it a crack again..